Marlin/configuration_store.cpp

```diff
@@ -64,28 +64,31 @@
  *  195  M206 XYZ  home_offset                      (float x3)
  *  207  M218 XYZ  hotend_offset                    (float x3 per additional hotend)
  *
- * Global Leveling:
+ * Global Leveling:                                 6 bytes
  *  219            z_fade_height                    (float)
+ *  223            GRID_MAX_POINTS_X                (uint8_t)
+ *  224            GRID_MAX_POINTS_Y                (uint8_t)
```

Comment:
I agree, but if we're going to sanity check ubl meshes, we need to store the size, mins, maxes, etc. along with the mesh (at the end of the eeprom) so there's something to check.  Right now, we're only storing the z_values, and there's no bounding or anything, so no way to know if what we read is an actual mesh or just a bunch of random data.

My logic in moving to one set of stored max x/y was that nothing is going to change unless you recompile and change something, and if you change bed leveling method, or grid size, all bets are off anyway, and everything has to be redone, generally (i.e., M502 -> M500).

But it's a cheap change (a few extra bytes) and I don't much care either way.  Was just trying to optimize.  I'll change it back momentarily.